managed-k8s/README.md

```diff
@@ -18,11 +18,38 @@ whereas each worker node has
 
 Provisioning time varies of course but we can expect to take around 10 minutes to spin up the cluster.
 
+## Retrieve kubeconfig
+
+Inside the cluster.tf you can see a defined output. the configured `value` needs
+to match the `gridscale_k8s` resource. In the given example, our `gridscale_k8s`
+resource is named `mycluster`:
+
+```
```

Comment:
I think you can add `hcl` here for GH markdown to render it.

Reply:
done